docs/gettingstarted/modfiles.md

```diff
@@ -248,3 +248,39 @@ An entry in `neoforge.mods.toml` does not need a corresponding `@Mod` annotation
 [versioning]: versioning.md
 [enumextension]: ../advanced/extensibleenums.md
 [featureflags]: ../advanced/featureflags.md
+## Mod files tree
+```
+root
+ |-- .gradle
+ |-- gradle
+ |-- build
+ |-- run
+ |-- src
+ |    |-- main
+ |    |    |-- java
+ |    |    |    |-- <com/net>
+ |    |    |    |    |-- <username>
+ |    |    |    |    |    |-- <modid>
+ |    |    |    |    |    |    |-- <some packages and single files>
+ |    |    |    |    |    |    |-- <modname>.java
+ |    |    |    |    |    |    |-- config.java
```

Comment:
We already have a section on this, but you cannot guarantee that all domains use a `com` or `net` extension. That is why the section specifies what to use.